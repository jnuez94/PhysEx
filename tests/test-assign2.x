string a;
string b;
string c;
string d;

void func simulation () {
	a = "printing a\n";
	b = "printing b\n";
	c = "printing c\n";
	d = "printing d\n";

	print(a);
	print(b);
	print(c);
	print(d);
}
