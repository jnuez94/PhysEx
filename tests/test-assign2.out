printing a
printing b
printing c
printing d
